tests/assembly/x86-return-float.rs

```diff
@@ -33,18 +33,18 @@ use minicore::*;
 // CHECK-LABEL: return_f32:
 #[no_mangle]
 pub fn return_f32(x: f32) -> f32 {
-    // CHECK: movl {{.*}}(%ebp), %eax
-    // CHECK-NOT: ax
+    // CHECK: movss {{.*}}(%ebp), %xmm0
+    // Make sure we do not move out of xmm0 again
+    // CHECK-NOT: xmm0
     // CHECK: retl
```

Comment:
If this works, it would also be an option and better due to enforcing more strictly... I _think_ it doesn't due to the frame pointers though. Maybe we should be forcing non-leaf frame-pointers instead...
```suggestion
    // CHECK: movss {{.*}}(%ebp), %xmm0
    // CHECK-NEXT: retl
```

Reply:
There's a `popl` in between the two; I can add a CHECK-NEXT for that as well. (same for the other test.)